tests/chainerx_tests/op_utils.py

```diff
@@ -291,15 +401,12 @@ def wrap(cls):
         else:
             classes = [(cls, cls.__module__)]
 
-        tests = [
-            ('forward', 'run_test_forward'),
-            ('backward', 'run_test_backward'),
-            ('double_backward', 'run_test_double_backward'),
-        ]
         for cls, mod in classes:
-            for func_suffix, method_name in tests:
-                _create_test_entry_function(
-                    cls, sys.modules[mod], devices, func_suffix, method_name)
+            if not issubclass(cls, OpTest):
+                raise TypeError(
+                    '@op_test decorator can only be applied to OpTest class '
+                    'definition.')
+            _create_test_entry_function(cls, sys.modules[mod], devices)
```

Comment:
Is this function name still appropriate? The logic is changed quite a bit.

Reply:
Yes, I think so.